ui/static/ui/js/utils.jsx

```diff
@@ -109,8 +109,13 @@ define("utils", ["jquery", "lodash", "reactaddons", "select2"],
       //this function can be used only if the component has been mounted
       var thiz = this;
       var isMultiTerms = this.props.multiple;
+
       //not allowing the clear for the multi terms dropdown
-      var allowClear = !isMultiTerms;
+      var allowClear;
+      if (this.props.allowClear === undefined) {
```

Comment:
So correct me if I am wrong, but as written, this conditional essentially sets a default, if a value is *not* passed into the component when it's created, so an else here would be redundant.